src/list.cpp

```diff
@@ -286,6 +287,8 @@ void list_probes(const BPFtrace &bpftrace, const std::string &search_input)
     std::cout << probe << std::endl;
   }
 
+  // kfuncs
+  bpftrace.btf_.display_funcs();
```

Comment:
Is there a function that kfuncs can trace but kprobe can't? If not, then I think showing parameters of kprobe func is straightforward. e.g., something like

```
% bpftrace -lv 'kprobe:vfs_open'
kprobe:vfs_open
    const struct path___2 * $path;
    struct file___2 * $file;
    int $ret;
```

(by the way, I'm not sure why trailing `___2` is added to some struct name...)

Reply:
yes, I have it like this in new version

Reply:
did you implement this?

Reply:
I thought we could unite kprobe and kfunc here, but I found that there was no BTF data for static function...
The new version looks good to me.